app/src/main/java/in/avimarine/boatangels/general/DialogUtils.java

```diff
@@ -0,0 +1,67 @@
+package in.avimarine.boatangels.general;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.view.View;
+import android.widget.Toast;
+import in.avimarine.boatangels.R;
+
+
+public class DialogUtils {
+
+    private static Toast toast;
+
+    public static void show(Context context, String message) {
```

Comment:
code duplication. I would write it this way:

    public static void show(Context context, String message) {
		showByLength(context, message, Toast.LENGTH_SHORT)
	}

	public static void showLong(Context context, String message) {
		showByLength(context, message, Toast.LENGTH_LONG)
	}
	
 public static showByLength(Context context, String message, int toastLength)
 {
          if (message == null) {
             return;
         }
         if (toast == null && context != null) {
             toast = Toast.makeText(context, message, toastLength);
         }
         if (toast != null) {
             toast.setText(message);
             toast.show();
         }
 }

Reply:
Fixed